src/main/java/com/minecolonies/inventory/GuiHandler.java

```diff
@@ -0,0 +1,34 @@
+package com.minecolonies.inventory;
+
+import com.minecolonies.colony.Field;
+import com.minecolonies.tileentities.ScarecrowTileEntity;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.network.IGuiHandler;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+/**
+ * Class which handles the GUI inventory.
+ */
+@SideOnly(Side.CLIENT)
+public class GuiHandler implements IGuiHandler
+{
+    @Override
+    public Object getServerGuiElement(int id, EntityPlayer player, World world, int x, int y, int z)
+    {
+        BlockPos            pos        = new BlockPos(x,y,z);
+        ScarecrowTileEntity tileEntity = (ScarecrowTileEntity) world.getTileEntity(pos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'tileEntity' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)